apollo/src/jsMain/kotlin/io/iohk/atala/prism/apollo/utils/HelperMethods.kt

```diff
@@ -0,0 +1,47 @@
+package io.iohk.atala.prism.apollo.utils
+
+/**
+ * The _require function is used to import modules in JavaScript.
+ *
+ * @param name The name of the module to import.
+ *
+ * @return The imported module.
+ */
+@Suppress("FunctionName", "UNUSED_PARAMETER")
+fun _require(name: String): dynamic = js("require(name)")
+
+/**
+ * A variable that represents the global object in JavaScript.
+ *
+ * The `global` variable is initialized lazily using the `lazy` delegate.
+ * It is assigned a dynamic value that depends on the current execution environment.
+ *
+ * In a Node.js environment, the `global` variable refers to the global object.
+ * In a browser environment, the `global` variable refers to the `window` object.
+ */
+val global: dynamic by lazy {
+    js("((typeof global !== 'undefined') ? global : self)")
```

Comment:
self is not contextually guaranteed, should be `window` for browsers
```suggestion
    js("((typeof global !== 'undefined') ? global : window)")
```

Reply:
But, In a Node.js environment, the `global` variable refers to the global object; in a browser environment, the `global` variable refers to the `window` object. That's why it should be kept as `self`. This way, in each environment, I would get the correct `global` variable.